example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
```

Comment:
Interesting. Why not call it something like "search-cache" in contrast to "bind-cache"?
(or app-cache?) ;-)